### PR TITLE
docs: add competition preparation checklist

### PR DESCRIPTION
**Pull request by texhnolyze**
# Summary
I've converted our current robot RoboCup checklist into rst.

## Checklist

- [x] Write documentation
- [x] Triage this PR and label it

### CODE REVIEW

**Comment by Flova** [comment]
image

File: bitbots_misc/bitbots_docs/docs/manual/testing/competition_preparation.rst [hunk 1]
```diff
@@ -0,0 +1,34 @@
+Robotics Competition Checklist
```
Comment (Flova) [inline bitbots_misc/bitbots_docs/docs/manual/testing/competition_preparation.rst:1]:
```suggestion
Hardware Checklist (Pre-Competition)
```

File: bitbots_misc/bitbots_docs/docs/manual/testing/competition_preparation.rst [hunk 2]
```diff
@@ -0,0 +1,34 @@
+Hardware Checklist (Pre-Competition)
+==============================
```
Comment (jaagut) [inline bitbots_misc/bitbots_docs/docs/manual/testing/competition_preparation.rst:2]:
```suggestion
====================================
```

File: bitbots_misc/bitbots_docs/docs/manual/testing/competition_preparation.rst [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+* Check if shoulders are bent
+* Inspect NUC power connectors
+
+Before Powering On
+------------------
+* Ensure arms and legs are in the correct configuration (team markers outside, cables loose not coiled)
+
+After Powering On
+-----------------
+* Verify ROS control torqueless and check robot model in RViz
```
Comment (jaagut) [inline bitbots_misc/bitbots_docs/docs/manual/testing/competition_preparation.rst:25]: We should provide the necessary commands...

File: bitbots_misc/bitbots_docs/docs/manual/testing/competition_preparation.rst [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+* Connect hands to legs while watching the robot mode in RViz
+* Check for motor communication issues during startup and afterwards
+* Run T-pose script
```
Comment (jaagut) [inline bitbots_misc/bitbots_docs/docs/manual/testing/competition_preparation.rst:28]: We should provide the necessary commands...

File: bitbots_misc/bitbots_docs/docs/manual/testing/competition_preparation.rst [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+* Test teleop walking
```
Comment (jaagut) [inline bitbots_misc/bitbots_docs/docs/manual/testing/competition_preparation.rst:29]: We should provide the necessary commands...

File: bitbots_misc/bitbots_docs/docs/manual/testing/competition_preparation.rst [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+* Test getting up
+* Verify robot-specific walking parameters
+* Perform extrinsic calibration
+* Check camera images for focus and proper transmission
```
Comment (jaagut) [inline bitbots_misc/bitbots_docs/docs/manual/testing/competition_preparation.rst:33]: We should provide the necessary commands...

**Review by jaagut — APPROVED** [review]

**Pull request merged**